stl/CMakeLists.txt

```diff
@@ -556,25 +603,27 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES ARCHIVE_OUTPUT_NAME "msvcprt${D_SUFFIX}")
 endfunction()
 
-add_stl_dlls("" "" "${VCLIBS_RELEASE_OPTIONS}" "/GL" "/LTCG;/opt:ref,icf")
-add_stl_dlls("d" "_DEBUG" "${VCLIBS_DEBUG_OPTIONS}" "" "/opt:ref,noicf")
+add_stl_dlls("" Release "/GL" "/LTCG;/opt:ref,icf")
+add_stl_dlls("d" Debug "${VCLIBS_DEBUG_OPTIONS}" "" "/opt:ref,noicf")
 
-function(add_stl_statics FLAVOR_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIONS)
+function(add_stl_statics FLAVOR_SUFFIX REL_OR_DBG ITERATOR_DEBUG_LEVEL)

```

Comment:
Suggestion: The single-character difference between the CMake parameter `ITERATOR_DEBUG_LEVEL` and the C++ macro `_ITERATOR_DEBUG_LEVEL` seems difficult to read. I'd recommend a more prominent change, e.g. `IDL_VALUE` or something.

(Unlike "benchmarking" vs. "benchmarks", I am less concerned about keeping the naming style the same at this layer, because the `_ITERATOR_DEBUG_LEVEL` spelling at the C++ level is extremely well-known to us, so when we see `IDL_VALUE` or whatever in CMake, we will know what it is ultimately intended for. The main danger I am worrying about is passing a *non*-underscored `ITERATOR_DEBUG_LEVEL` to C++, which would be silently ignored.)

Reply:
SGTM